rules/apps.libsonnet

```
@@ -200,6 +200,20 @@
               workload_type: 'statefulset',
             },
           },
+          {
+            record: 'namespace_workload_pod:kube_pod_owner:relabel',
+            expr: |||
+              max by (%(clusterLabel)s, namespace, workload, pod) (
+                label_replace(
+                  kube_pod_owner{%(kubeStateMetricsSelector)s, owner_kind="Job"},
+                  "workload", "$1", "owner_name", "(.*)"
+                )
+              )
+            ||| % $._config,
+            labels: {
+              workload_type: 'Job',
```

```suggestion
              workload_type: 'job',
```

let's keep this aligned with other `workload_type` labels.